gradle/libs.versions.toml

```
@@ -53,12 +53,10 @@ hilt = "2.55"
 hiltNavigationCompose = "1.2.0"
 
 ## Network
-# okhttp
-# # https://square.github.io/okhttp/
-okhttp = "4.12.0"
-# Retrofit
-# # https://github.com/square/retrofit
-retrofit = "2.11.0"
+# https://github.com/Foso/Ktorfit
+ktorfit = "2.1.0"
+# https://github.com/ktorio/ktor
+ktorVersion = "2.3.12"
```

```suggestion
ktor = "2.3.12"
```
(대왕소소) 이렇게 바꿔보는건 어떨까요?